### PR TITLE
fix a few cases in which some content was not showing up due to allOf references

### PR DESCRIPTION
**Pull request by danieleds**
Resolves `allOf` references in the schema. Otherwise, I found that in some special cases the request body was missing.

### CODE REVIEW

**Review by bourdakos1 — APPROVED** [review]
lgtm

**Comment by danieleds** [comment]
Cool! I can't merge it myself so I'll leave it to you

**Pull request merged**
